src/components/trade-button/index.tsx

```diff
@@ -29,6 +29,7 @@ export const TradeButton = (props: TradeButtonProps) => {
       height='54px'
       w='100%'
       onClick={props.onClick}
+      className={clsx(background)}
```

Comment:
Changed to tailwind config - as otherwise chakra's background config seems to be overriden.